osmomath/mul_coins_with_number_test.go

```diff
@@ -0,0 +1,36 @@
+package osmomath
+
+import (
+	"fmt"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMulCoinAmtsByU64(t *testing.T) {
+	type testcase struct {
+		i    sdk.Int
+		u    uint64
+		want sdk.Int
+	}
+	tests := map[string]testcase{
+		"mul with zero": {
+			sdk.NewInt(5), 0, sdk.ZeroInt(),
+		},
+		"5 * 1": {
+			sdk.NewInt(5), 1, sdk.NewInt(5),
+		},
+		"5 * 10": {
+			sdk.NewInt(5), 10, sdk.NewInt(50),
+		},
+	}
+
+	for name, tt := range tests {
+		fmt.Println("Start")
```

Comment:
TODO : erase this

Reply:
thanks for your reply. This code can be used for testing the cases relevant to the coin first sir